manta-trusted-setup/src/groth16/kzg.rs

```diff
@@ -384,7 +384,7 @@ where
 }
 
 /// Contribution Accumulator
-#[derive(Clone, Debug, Eq, Hash, PartialEq)]
+#[derive(CanonicalDeserialize, CanonicalSerialize, Clone, Debug, Eq, Hash, PartialEq)]
```

Comment:
I commented out the `impl CanonicalSerialize for Accumulator` in this file because I was having trouble with `Serializer` for the PPoT configuration